Chapter19/02_play.py

```diff
@@ -38,6 +39,7 @@
         mu_v = net(obs_v)
         action = mu_v.squeeze(dim=0).data.numpy()
         action = np.clip(action, -1, 1)
+        if np.isscalar(action): action = [action]
```

Comment:
I'd prefer to stick to pip8. So, this block should look like
```
if np.isscalar(action):
    action = [action]
```

Or, if you prefer to have it one-liner, this one:
```
action = [action] if np.isscalar(action) else action
```

Reply:
I agree, and have committed PEP8-compliant version.